.github/workflows/tests.yml

```diff
@@ -2,6 +2,29 @@ name: Run tests with workflow_call
 
 on:
   workflow_call:
+    inputs:
+      fluence-env:
+        description: "Fluence enviroment to run tests agains"
+        type: string
+        default: "local"
+      nox-image:
+        description: "nox image tag"
+        type: string
+        default: "fluencelabs/nox:unstable_minimal"
+      ref:
+        description: "GitHub ref to checkout to"
+        type: string
+        default: "main"
+      flox-version:
+        description: "@fluencelabs/flox version"
```

Comment:
it's not flox, it's `fluencelabs/fluence-cli`

Reply:
@nahsi

Reply:
This is the remains from fcli renaming to flox. I should change it back, but maybe do it later everywhere in a separate PR? 🙏 This is just cosmetics after all